components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -746,7 +746,7 @@ <h1 class="can-collapse closed" data-name="tags">
             <!-- MAP ANNOTATIONS -->
             <div id="mapAnnsPane">
             <h1 class="can-collapse closed" data-name="keyvaluepairs">
-                Key-Value Pairs
+                Attributes
```

Comment:
Key-Value Pairs was the term we agreed on for both clients.
I would not change this without a good reason, since we'd also need to update Insight, help etc: http://help.openmicroscopy.org/managing-data.html#keyvalue